scripts/run_performance_job.py

```diff
@@ -312,11 +667,15 @@ def run_performance_job(args: RunPerformanceJobArgs):
         else:
             runtime_id = f"linux-{args.architecture}"
 
+        dotnet_executable_path = os.path.join(ci_setup_arguments.install_dir, "dotnet")
+
+        os.environ["MSBUILDDISABLENODEREUSE"] = "1" # without this, MSbuild will be kept alive
+
         # build Startup
         RunCommand([
```

Comment:
May be worth adding a binlog output here, something like from copied below: `f"/bl:{os.path.join(args.performance_repo_dir, 'artifacts', 'log', build_config, 'StartupBuild.binlog')}"]`. This would help with debugging why any build failures are occuring if we see them.

Reply:
This expands to all of the build steps at this part of the script. This feedback comes from the first run here: https://dev.azure.com/dnceng-public/public/_build/results?buildId=552028&view=results having all net8.0 runs fail but not having any useful information about why they failed. Although rerunning the tests seems to have fixed that issue.